docs/book/how-to/stack-deployment/deploy-a-cloud-stack-with-terraform.md

```diff
@@ -0,0 +1,337 @@
+---
+description: Deploy a cloud stack using Terraform
+---
+
+# Deploy a cloud stack with Terraform
+
+ZenML maintains a collection of [Terraform modules](https://registry.terraform.io/modules/zenml-io/zenml-stack)
+designed to streamline the provisioning of cloud resources and seamlessly
+integrate them with ZenML Stacks. These modules simplify the setup process,
+allowing users to quickly provision cloud resources as well as configure and
+authorize ZenML to utilize them for running pipelines and other AI/ML operations.
+
+By leveraging these Terraform modules, users can ensure a more efficient and
+scalable deployment of their machine learning infrastructure, ultimately
+enhancing their development and operational workflows. The modules
+implementation can also be used as a reference for creating custom Terraform
+configurations tailored to specific cloud environments and requirements.
+
+{% hint style="info" %}
+Terraform requires you to manage your infrastructure as code yourself. Among
+other things, this means that you will need to have Terraform installed on your
+machine and you will need to manually manage the state of your infrastructure.
+
+If you prefer a more automated approach, you can use [the 1-click stack deployment feature](deploy-a-cloud-stack.md)
+to deploy a cloud stack with ZenML with minimal knowledge of Terraform or cloud
+infrastructure for that matter.
+
+If you have the required infrastructure pieces already deployed on your cloud, 
+you can also use [the stack wizard to seamlessly register your stack](register-a-cloud-stack.md).
+{% endhint %}
```

Comment:
```suggestion
you can also use [the stack wizard to seamlessly register your stack](register-a-cloud-stack.md).
{% endhint %}
```

I'm not sure that these Markdown links show up as links when inside these hints... I think you'll have to use an HTML <a> tag instead, but test it out in gitbook preview to confirm..

Reply:
you were right about this. I need to use relative links. Absolute links don't work here.

Reply:
updated and verified that all links work now